tools/lint/lint.py

```diff
@@ -655,6 +655,16 @@ def incorrect_path(script, src):
         if incorrect_path("testdriver-vendor.js", src):
             errors.append(rules.TestdriverVendorPath.error(path))
 
+        script_path = None
```

Comment:
This code checks for unnecessary reftest-wait.js, but what about 
takeScreenshot without reftest-wait?

Reply:
It's checking for reftest-wait.js without reftest-wait, primarilly to find cases where people forgot to add the class, not cases where people included the script needlessly (although there was one of those). Do we have examples of people trying to call takeScreenshot without including the script? That seems less likely.

Reply:
Ok. I think the to_fix field is incorrect in the other file then, suggested
a rewording.